cachi2/core/models/sbom.py

```diff
@@ -104,3 +115,575 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self, doc_namespace: str) -> "SPDXSbom":
+        """Convert a CycloneDX SBOM to an SPDX SBOM.
+
+        Args:
+            doc_namespace: SPDX document namespace. Namespace is URI of indicating
+
+        """
+        packages = []
+        relationships = []
+
+        packages.append(
+            SPDXPackage(
+                name="",
+                versionInfo="",
+                SPDXID="SPDXRef-DocumentRoot-File-",
+            )
+        )
+
+        for component in self.components:
+            annotations = []
+            for prop in component.properties:
+                annotations.append(
+                    SPDXPackageAnnotation(
+                        annotator="Tool:cachi2:jsonencoded",
```

Comment:
```suggestion
                        annotator="Tool: cachi2:jsonencoded",
```
Minor detail for consistency with the other `Tool: ` strings